Stratis.Bitcoin.Features.RPC/RPCFeature.cs

```diff
@@ -37,7 +37,8 @@ public override void Start()
                 // TODO: The web host wants to create IServiceProvider, so build (but not start) 
                 // earlier, if you want to use dependency injection elsewhere
                 this.fullNode.RPCHost = new WebHostBuilder()
-                .UseLoggerFactory(this.nodeSettings.LoggerFactory)
+                // ConfigureServices (below) gets this from the full node
```

Comment:
if this is not needed, please remove the comment

Reply:
done